### ISSUE TITLE
A false negative about the rule FinalFieldCouldBeStatic

### ISSUE THREAD

**Issue by Belle-PL** [pmd/pmd#3949]
**Affects PMD Version:6.42.0** 

**Rule:FinalFieldCouldBeStatic**

Please provide the rule name and a link to the rule documentation:
<https://pmd.github.io/latest/pmd_rules_java_design.html#finalfieldcouldbestatic>

**Description:**
Hi, I found a false negative about the rule FinalFieldCouldBeStatic. Please read the example below. PMD should have reported a warning in this line, but it doesn't. I think this is caused by the ignorance of parenthesized literal. Thanks for your consideration.

**Code Sample demonstrating the issue:**
```java
public final int BAR = (42);
```

**Expected outcome:**

PMD should report a violation at line 1, but doesn't. This is a false-negative.

**Running PMD through:** *[Maven]*

**Comment by dalizi007** [comment]
Please give me the subject
